src/Auth0/Login/Auth0Service.php

```diff
@@ -37,7 +38,7 @@ class Auth0Service
     public function __construct(
         array $auth0Config = null,
         StoreInterface $sessionStorage = null,
-        SessionStateHandler $sessionStateHandler = null
+        StateHandler $stateHandler = null
```

Comment:
We were going to remove the default checking here, correct? Would it be common for a developer to call this directly?

Reply:
If I understand your question correctly it's about removing the `= null` from the arguments? 

I'd love to make these argument required in the future and remove the fallback behaviour in the constructor. Maybe this is something we can do in the 7.0.0-dev branch?

Reply:
Alright, that makes sense. 👍